_includes/js/custom.js

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+document.addEventListener('DOMContentLoaded', () => {
+  let successes = document.body.getElementsByClassName('success-table-element');
```

Comment:
these `let`s can and should be `const` (https://google.github.io/styleguide/jsguide.html#features-use-const-and-let)

Reply:
I changed the spacing to make it conform to the style guide with the wrapping the `const`s caused, but somehow lost the `const`s along the way...